finance/server/src/main/java/com/google/finapp/SpannerDaoPGImpl.java

```diff
@@ -0,0 +1,367 @@
+// Copyright 2021 Google LLC
```

Comment:
nit: Is this still the right header - year 2022.

Reply:
I'd suggest leaving this and having a separate PR for updating years. Thoughts @voulg?

Reply:
Usually these headers are based on when the file was added. I agree this should be 2022 for all new files.

Reply:
👍

Reply:
This has been addressed in new commit